crates/javy/src/config.rs

```diff
@@ -38,6 +38,7 @@ bitflags! {
     pub(crate) struct JavyIntrinsics: u32 {
         const STREAM_IO = 1;
         const JSON = 1 << 1;
+        const CRYPTOX = 1 << 2;
```

Comment:
Similar to above, given that this API is not namespaced through `Javy`, it might make sense to add it as part of `JSIntrinsics` instead, similar to `TEXT_ENCODING` (here https://github.com/bytecodealliance/javy/pull/685/files#diff-f9510ffe1a14a724c735bc8f7f3d0196470ccf82f09f399287be0db59d593cc6R21) -- that is, if we decide on start standardizing this API through Winter CG as discussed offline.